torch/csrc/jit/tracer.cpp

```diff
@@ -35,6 +36,14 @@ std::string getPythonInterpreterStackTrace() {
   }
   return stack_trace.str();
 }
+#else
+std::string torch::jit::tracer::getPythonInterpreterStackTrace() {
+  return std::string();
+}
```

Comment:
I think it would be cleaner to ifdef out the use of this function instead of defining the stub

Reply:
Done